lib/aggregators.js

```diff
@@ -16,7 +16,7 @@ Aggregator.prototype.makeBufferKey = function(key, tags) {
 
 Aggregator.prototype.addPoint = function(Type, key, value, tags, host, timestampInMillis, options) {
     const bufferKey = this.makeBufferKey(key, tags);
-    if (!this.buffer.hasOwnProperty(bufferKey)) {
+    if (!Object.prototype.hasOwnProperty.call(this.buffer, bufferKey)) {
```

Comment:
We can turn off this check, but it’s on by default in ESLint for good reason: you can’t actually count on all objects having the `hasOwnProperty()` method. (There’s a newer, safe tool for this `Object.hasOwn(this.buffer, bufferKey)`, but it’s new in Node.js 16, so can’t be used in this project.)

*Technically* we could turn off this rule since we are probably always in control of the creation of `this.buffer` and can make sure it has a `hasOwnProperty` function. But this is good practice. Other alternatives here are:
1. Create the buffer with `this.buffer = Object.create(null)`, which will result in an object that is missing `hasOwnProperty()`, but is also missing any of the things we are doing the own property check for in the first place. Then we could just drop the `if`.
2. Use a `Map` instead of a simple object. It was made specifically for the sort of use-case that’s happening here, and is supported in versions of Node.js this package supports.